gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -144,6 +150,16 @@
           "metadata");
 
   private static final String LIST_OBJECT_FIELDS_FORMAT = "items(%s),prefixes,nextPageToken";
+  private static final String METHOD_CREATE_BUCKET = "createBucket";
```

Comment:
Each of these constants used in a single method - not sure that it worth defining class-level constants for this.

Reply:
Fair enough, makes sense.  I will revert them